packages/shared/src/components/HorizontalScroll/useHorizontalScrollHeader.tsx

```diff
@@ -1,38 +1,54 @@
 import React, {
```

Comment:
Most of the returned values from this hook are not memoized, we should do it as consuming components could have something from this in the deps list.

Reply:
Yes. Will work on this one after the public squads request clean up.